config/src/config/logger_config.rs

```diff
@@ -31,25 +30,24 @@ impl Default for LoggerConfig {
             enable_backtrace: false,
             is_async: true,
             level: Level::Info,
-            // This is the default port used by tokio-console
-            // setting console_port to None will disable tokio console even if aptos-console
-            // feature is enabled
-            console_port: Some(6669),
             enable_telemetry_remote_log: true,
             enable_telemetry_flush: true,
             telemetry_level: Level::Error,
+
+            // This is the default port used by tokio-console.
+            // Setting this to None will disable tokio-console
+            // even if the "tokio-console" feature is enabled.
+            tokio_console_port: None,
```

Comment:
Hmm, where do we set this to `Some(..)` now? There is `randomize_ports()` fn below but it doesn't appear to be called.

Reply:
This should only be set to `Some(...)` if we actually want to enable it, right? Are there any tests that actually use it? If so, that's (great) news to me 😄 My reasoning for `None` was that configs should default to production, and if folks want to test/experiment, they should override them 🤔

Reply:
True, that is reasonable; we want to set a port only when we enable it. But, I was thinking we should set a default port when the feature is actually enabled, so no code is change is necessary to enable tokio-console. Otherwise, it might be too much friction to figure this out for someone.

Reply:
> But, I was thinking we should set a default port when the feature is actually enabled, so no code is change is necessary to enable tokio-console.

Aah, yeah, that makes sense. FWIW, this is something that should hopefully be enabled soon (i.e., I'm currently working on a way to add finer-grained defaults, such that if a compiler feature is enabled (or a node is a certain type), we can specify these kinds of defaults. It'll hopefully be built on top of this: https://github.com/aptos-labs/aptos-core/pull/7783.